pennylane/pulse/transmon.py

```diff
@@ -390,14 +397,12 @@ def qnode(params):
     # Note that exp(-iw)a* + exp(iw)a = cos(w)X - sin(w)Y for a=1/2(X+iY)
     # We compute the `coeffs` and `observables` of the EM field
     coeffs = [
-        AmplitudeAndPhaseAndFreq(qml.math.cos, amplitude, phase, freq),
         AmplitudeAndPhaseAndFreq(qml.math.sin, amplitude, phase, freq),
```

Comment:
This should make black put it in one line :)
```suggestion
        AmplitudeAndPhaseAndFreq(qml.math.sin, amplitude, phase, freq)
```